packages/flutter/test/material/text_field_focus_test.dart

```diff
@@ -130,7 +132,7 @@ void main() {
     await tester.pumpWidget(Container());
 
     expect(tester.testTextInput.isVisible, isFalse);
-  });
+  }, skip: Platform.isMacOS);
```

Comment:
`skip: true` until we deflake it?

Reply:
Sounds good, added a link to the bug too